auto.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/expect -f
+ 
+set timeout -1
+spawn ./run.sh
+expect "*Input row and column:\r" 
+send -- "0 29\r"
+expect "*Input row and column:\r" 
+send -- "0 0\r"
+expect "*Input row and column:\r" 
+send -- "0 28\r"
+expect "*Input row and column:\r" 
+send -- "1 21\r"
+expect "*Input row and column:\r" 
+send -- "1 0\r"
+
+expect eof
```

Comment:
Niezły pomysł, natomiast po 3 tygodniach jak tu wrócisz i będziesz to czytał to będziesz się nielicho zastanawiał co właściwie to robi. Jakiś komentarz czy linijka z `echo` nie byłaby głupia.

Reply:
Dodałem dwa komentarze. W kolejnych migawkach test będzie się zmieniał, więc komentarze również